docs/getting_started/prepare_environment.md

```diff
@@ -32,6 +32,26 @@ sudo usermod -aG docker $USER
 
 **You will need to log out and lock back in** for user changes to take effect.
 
+You can also setup this environment on CBL-Mariner
+```bash
+# Install required dependencies.
+sudo dnf -y install git make tar wget curl rpm golang genisoimage python bison gawk glibc-devel binutils kernel-headers
+```
+You may want to install rpmlint, in this case you need to add the extended repo in /etc/yum.repos.d/ as mariner-extended.repo
+```bash
+[mariner-official-extended]
```

Comment:
We publish this file via the `mariner-repos-extended` package (`sudo tdnf -y install mariner-repos-extended`). 

Unless there is a need to edit the repo file that should work instead of manually adding the repo.

Reply:
Good suggestion. Corrected.